charts/kyverno/templates/_helpers.tpl

```diff
@@ -116,3 +116,26 @@ maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
 {{- define "kyverno.imagePullSecret" }}
 {{- printf "{\"auths\":{\"%s\":{\"auth\":\"%s\"}}}" .registry (printf "%s:%s" .username .password | b64enc) | b64enc }}
 {{- end }}
+
+
+{{- define "kyverno.resourceFilters" -}}
+{{- $resourceFilters := .Values.config.resourceFilters }}
+{{- if .Values.excludeKyvernoNamespace }}
+  {{- $resourceFilters = prepend .Values.config.resourceFilters (printf "[*,%s,*]" (include "kyverno.namespace" .)) }}
+{{- end }}
+{{- tpl (join "" $resourceFilters) . }}
+{{- end }}
+
+{{- define "kyverno.webhooks" -}}
+{{- $excludeDefault := dict "key" "kubernetes.io/metadata.name" "operator" "NotIn" "values" (list (include "kyverno.namespace" .)) }}
+{{- $newWebhook := list }}
+{{- range $webhook := .Values.config.webhooks }}
```

Comment:
Shouldn't it be `{{- range $_, $webhook := .Values.config.webhooks }}` ?

Reply:
I think for lists, does work with only 1 variable, at least that's working at this time.  I  don't think it behaves like Go where range of slice have to handle index as first variable.  

I've been testing with these and both render correctly:

```
helm template kyverno charts/kyverno
helm template kyverno charts/kyverno --values charts/kyverno/ci/webhooks-values.yaml
```

I did verify doing two variable approach also works but doesn't seem to be necessary like it is with Go

Reply:
:+1: